src/qt/walletmodel.cpp

```diff
@@ -336,11 +331,6 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
 {
     QByteArray transaction_array; /* store serialized transaction */
 
-    if(isAnonymizeOnlyUnlocked())
-    {
-        return AnonymizeOnlyUnlocked;
-    }
-
```

Comment:
Same comment as above.  Maybe this is ensured some other way which I haven't seen but I would think a final check here would be a good idea even so.